starter-code/routes/index.js

```diff
@@ -1,9 +1,25 @@
-const express = require('express');
-const router  = express.Router();
+const express = require("express");
+const router = express.Router();
+// const multer = require("multer");
```

Comment:
You need to import multer, otherwise it will later throw you an error "multer is not defined"